src/chainstate/stacks/db/mod.rs

```diff
@@ -258,16 +258,16 @@ impl<'a> ClarityTx<'a> {
 
     #[cfg(test)]
     pub fn commit_block(self) -> () {
-        self.block.commit_block()
+        self.block.commit_block();
     }
 
     pub fn commit_block_will_move(self, will_move: &str) -> () {
-        self.block.commit_block_will_move(will_move)
+        self.block.commit_block_will_move(will_move);
```

Comment:
Probably, but I try not to make too many stylistic changes while implementing other features. If we want to unify the coding style of the codebase (which I'm in favor of), we should add a linter to the circle tests and open a PR to specifically address linting. Otherwise, we could end up with a lot of spurious conflicts between PRs.